tools/topology/topology2/cavs-mixin-mixout-hda.conf

```diff
@@ -113,9 +113,27 @@ Object.Pipeline {
 			Object.Widget.host-copier.1 {
 				stream_name $ANALOG_PLAYBACK_PCM
 				pcm_id 0
-				num_input_audio_formats 3
+				num_input_audio_formats 6
 				num_output_audio_formats 1
 				Object.Base.input_audio_format [
+					{
```

Comment:
Modify instead nocodec topology to enable testing with these formats. No need to have this in HDA generic.